web-client/src/persistence/s3/uploadDocumentFromClient.ts

```diff
@@ -17,7 +17,7 @@ export const uploadDocumentFromClient = async ({
 }: {
   applicationContext;
   document: any;
-  key: string;
+  key?: string;
```

Comment:
Why was this made optional ?

Reply:
The function is responsible for generating a new document id if a 'key' isn't passed in. [Line 23](https://github.com/flexion/ef-cms/blob/9e59f1dcc1aa78e2aa455177a7d528233714af7e/web-client/src/persistence/s3/uploadDocumentFromClient.ts#L23)